Makefile

```diff
@@ -106,28 +106,30 @@ unit-test: vendor
 docker_stop:
 	-docker ps -a --format '{{.Names}}' | grep kraken | while read n; do docker rm -f $$n; done
 
+env:
```

Comment:
Most things ive seen online call it `venv`, and i think i prefer that too. Up to you.

Reply:
Also you should add `requirements-tests.txt` as a dependency for that target.